memcached.c

```diff
@@ -4232,6 +4249,7 @@ static void usage(void) {
     verify_default("ssl_keyformat", settings.ssl_keyformat == SSL_FILETYPE_PEM);
     verify_default("ssl_verify_mode", settings.ssl_verify_mode == SSL_VERIFY_NONE);
 #endif
+    printf("-N, --napi_ids            max number of napi ids for NAPI ID based thread selection\n");
```

Comment:
add a "see doc/napi_ids.txt for detail" or similar? Also saying NAPI twice is kinda silly. "max number of ids for NAPI ID based thread" :)

Reply:
ok